lib/diagnostic.js

```diff
@@ -15,9 +15,18 @@
 
 const fs = require('fs'); // jshint ignore: line
 
-const sumLines = (filename, callback) => {
-  /* your response here */
-  callback();
+let fileName = process.argv[2];
```

Comment:
This would work if we were running this function from the command line. But that's not the case here. This is function that will be called with a `.txt` file and a callback.